Mage.Client/pom.xml

```diff
@@ -55,9 +55,9 @@
             <version>0.8.5</version>
         </dependency>
         <dependency>
-            <groupId>com.google.collections</groupId>
-            <artifactId>google-collections</artifactId>
-            <version>1.0</version>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+            <version>20.0</version>
```

Comment:
I just made this match the guava version that was being pulled in as a dependency for a different pom.